CONTRIBUTING.md

```diff
@@ -1,24 +1,56 @@
-# Contributing
+## Contributing
 
-We welcome any and all contributions. To get started,
+We're excited to have you contribute. Here's a step-by-step guide to help you get started.
 
-1. Make sure you have `yarn` installed. We are still using version 1.x. The installation instructions can be found here: https://classic.yarnpkg.com/lang/en/docs/install
-2. [Fork](https://github.com/alchemyplatform/aa-sdk/fork) this repo
-3. Run `yarn` to install dependencies
-4. Make changes to packages
+### Getting Started
 
-## Pull Request Checklist
+1. **Fork and Clone**: First, [fork aa-sdk repository](https://github.com/alchemyplatform/aa-sdk/fork). Then, clone your forked repo to your local machine.
 
-- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
-- [ ] Did you update relevant docs? (docs are found in the `site` folder, see guidleines below)
-- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
-- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
-- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples? (e.g. `feat!: breaking change`)
-- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:fix`)
-- [ ] Is the base branch you're merging into `development` and not `main`?
+2. **Install Dependencies**: Ensure you have `yarn` installed (we use version 1.x). Installation instructions are available [here](https://classic.yarnpkg.com/lang/en/docs/install). Run `yarn` in the project root to install all necessary dependencies.
 
-## Docs guidelines
+3. **Testing Environment**: Before making changes, make sure to verify the testing environment.
 
-We leverage `vitepress` to build our docs. You can find the docs in the `site` folder. To run the docs locally, run `yarn docs:dev` from the `site` folder. To build the docs, run `yarn docs:build`. Docs are automatically deployed with each package published.
+   - Use the Node version specified in `package.json` (currently 18.10). Run `node -v` to check your version.
```

Comment:
I think @avasisht23 made this 18.16

Reply:
![image](https://github.com/alchemyplatform/aa-sdk/assets/83442423/8b95bed4-9d0a-4be3-9c7d-cbc461b4ce45)

Reply:
I'm changing this to 18.16 in an upcoming branch, but sounds good for now.

Reply:
changed!